### PR TITLE
Fix editor tests on Windows

### PR DESCRIPTION
**Pull request by calixteman**
- In #15373, we implemented copy/paste actions in using the system clipboard.
For any reasons, on Windows, the clipboard doesn't contain the expected data when the tests are ran in parallel, hence the tests which are using the clipboard need to be ran sequentially.
- Make sure that we can paste after having copied.

### CODE REVIEW

File: test/integration/test_utils.js [hunk 1]
```diff
@@ -87,3 +87,20 @@ function getSelectedEditors(page) {
...
+async function waitForEvent(page, eventName, timeout = 30000) {
+  await Promise.race([
+    // add event listener and wait for event to fire before returning
+    page.evaluate(name => {
+      return new Promise(resolve => {
+        const listener = e => {
+          document.removeEventListener(name, listener);
+          resolve();
+        };
+        document.addEventListener(name, listener);
```
Comment (Snuffleupagus) [inline test/integration/test_utils.js:100]: Unless I'm missing something, could this not be simplified to only the following?
```suggestion
        document.addEventListener(name, resolve, { once: true });
```

**Review by Snuffleupagus — APPROVED** [review]
r=me, with one suggestion and with passing tests; thank you!

File: test/integration/test_utils.js (hunk 1 above)
Comment (calixteman) [reply test/integration/test_utils.js]: I forgot this argument.

**Comment by calixteman** [comment]
I think windows tests are in a better shape with this patch.
We still have this issue with PageOpen and an intermittent with zip codes, but at least it should fix the editing ones.
I'm not super happy to add these `waitForTimeout` but I don't see any other way to do.
@Snuffleupagus, are you still ok with this patch (I made few changes after your approval) ?

**Comment by Snuffleupagus** [comment]
> I think windows tests are in a better shape with this patch.

Agreed, it seems like it helped on the bots. (It's weird that these problems don't show up locally...)

> We still have this issue with PageOpen

Do you have time to continue investigating this in #15308?

> I'm not super happy to add these `waitForTimeout` but I don't see any other way to do.

Agreed, but since this is only in tests I suppose it's not that bad overall.

> are you still ok with this patch (I made few changes after your approval) ?

Sure (I looked at the changes and it seems fine).

**Comment by calixteman** [comment]
> > We still have this issue with PageOpen
> 
> Do you have time to continue investigating this in #15308?
> 
I'll try to find some time this week or the week after.

**Pull request merged**